packages/ap-core/alliance_platform/core/auth.py

```diff
@@ -0,0 +1,76 @@
+from alliance_platform.core.settings import ap_core_settings
+from django.apps import AppConfig
+from django.db.models import Model
+
+
+def default_resolve_perm_name(
+    app_config: AppConfig,
```

Comment:
The main reason I didn't do this is because this is just migrating our existing function, and it's the function interface expected by csvpermissions. The `default_resolve_perm_name` isn't really designed to be called directly, but I'll add it to the `app_config` param desc to describe how to pass it when passing a `model`. The `resolve_perm_name` method (the one you would be calling) gives examples of this.

I'll look into maybe supporting an overloaded interface so `app_config` can be omitted if `model` is supplied

Reply:
I've made `resolve_perm_name` accept either AppConfig or Model and defined overloads for it for typing. 

`default_resolve_perm_name` still accepts both, but `resolve_perm_name` will extract it from the model for you